.github/workflows/release.yml

```diff
@@ -31,7 +31,7 @@ jobs:
           server-password: MAVEN_CENTRAL_TOKEN # env variable for token in deploy
 
       - name: Release package
-        run:  mvn -B deploy
+        run:  mvn -B deploy -DskipTests
```

Comment:
Is this wise? I guess you're relying on tests having run when the PR was pushed/merged? But it would be perfectly possible to push a tag to master and create a release and bypass those checks??

Reply:
The release come from the master, basically it's a snapshot from master without changes and master passed the `check integrity` never we will do a release directly from a feature or any other branch without merge it first on master.
The release process -> branch/feature -> merge on master -> release branch -> create release -> publish release -> merge release branch on master for the next development iteration.